src/System.CommandLine.Tests/ArgumentTests.cs

```diff
@@ -333,9 +339,12 @@ public void Command_ArgumentResult_Parent_is_set_correctly_when_token_is_implici
 
                 argumentResult
                     .Parent
-                    .Symbol
                     .Should()
-                    .Be(command);
+                    .BeAssignableTo<CommandResult>()
```

Comment:
I think you can use `BeOfType` here, no?

Reply:
iirc it's `RootCommandResult` here (which is internal)